File: example-java/src/main/java/com/github/simonpercic/oklogexamplejava/data/api/okhttp3/RestApi3.java
package com.github.simonpercic.oklogexamplejava.data.api.okhttp3;

import com.github.simonpercic.oklogexamplejava.data.api.model.response.show.ShowResponse;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * @author Simon Percic <a href="https://github.com/simonpercic">https://github.com/simonpercic</a>
 */
public interface RestApi3 {

    @GET("/shows")
    Call<List<ShowResponse>> getShows();
}

File: example-kotlin/src/main/kotlin/com/github/simonpercic/oklogexamplekotlin/data/api/okhttp3/RestApi3.kt
package com.github.simonpercic.oklogexamplekotlin.data.api.okhttp3

import com.github.simonpercic.oklogexamplekotlin.data.api.model.response.show.ShowResponse

import retrofit2.Call
import retrofit2.http.GET

/**
 * @author Simon Percic [https://github.com/simonpercic](https://github.com/simonpercic)
 */
interface RestApi3 {

    @GET("/shows")
    fun getShows(): Call<List<ShowResponse>>
}
